tests/installation/partitioning_warnings.pm

```diff
@@ -179,7 +179,7 @@ sub run {
     else {
         ## Clean up previous root
         remove_partition;
-        addboot if get_var('UEFI');
+        addboot if (get_var('UEFI') or get_var('OFW'));
```

Comment:
Better use `||` insted of `or`.

Reply:
why?

Reply:
Because in the codebase `||` is used nearly everywhere.

Reply:
I think the precedence does not matter in this case. WDYT ?

Reply:
It doesn't matter - it's just a cosmetic detail ;)

Reply:
updated :)